conductr_cli/docker_machine.py

```diff
@@ -1,9 +0,0 @@
-import os
-
-DEFAULT_DOCKER_MACHINE_NAME = 'default'
-DEFAULT_DOCKER_MACHINE_RAM_SIZE = '3072'
-DEFAULT_DOCKER_MACHINE_CPU_COUNT = '4'
```

Comment:
I've retained these constants as they could indeed be different for docker-machine.